Content.Server/_Moffstation/StationEvents/Components/VentCrittersRuleComponent.cs

```diff
@@ -23,5 +22,18 @@ public sealed partial class VentCrittersRuleComponent : Component
     [DataField]
     public int SpawnAttempts = 100;
 
-    public EntityCoordinates? Location;
+    [DataField]
+    public TimeSpan NextPopup;
+
+    [DataField]
+    public TimeSpan PopupDelay = TimeSpan.FromSeconds(5);
+
+    [DataField]
+    public EntityUid? Location;
```

Comment:
```suggestion
    [ViewVariables]
    public EntityUid? Location;
```
This isn't part of the rule definition, so it shouldn't be a field.